index.html

```diff
@@ -336,7 +336,7 @@
   <section id="abstract">
   <p>
   This document describes a formal model and a common representation 
-  for a Web of Things (WoT) Thing Description. 
+  for a Web of Things (WoT) Thing Description 1.1. 
```

Comment:
Not sure if this change is actually necessary since the version information come later

Reply:
I think, it is important to point out that this document is about TD 1.1. This is also provided by JSON-LD 1.1 https://www.w3.org/TR/json-ld11/#abstract